custom-plugins/fictive_studios/admin/models/api/models-api.php

```diff
@@ -0,0 +1,47 @@
+<?php
+namespace FictiveCodes;
+
+class ModelsAPIAdmin
+{
+    public function get_models_data()
```

Comment:
I do not think we need an api that fetches all the details. 
i think we need an API `get_models_data_by_id` . 
every other place we would just need the model details.

Reply:
I have revised the 'Get All Models' API to provide only the basic details of all models. Additionally, I have implemented another API endpoint to retrieve model data by its ID, which returns all associated details of that specific model.